cvat/apps/engine/views.py

```diff
@@ -616,10 +614,32 @@ def data(self, request, pk):
                     status=status.HTTP_400_BAD_REQUEST)
             serializer = DataSerializer(data=request.data)
             serializer.is_valid(raise_exception=True)
+            data = {k: v for k, v in serializer.validated_data.items()}
+
+            tmp_path = os.path.join('./data/tmp', pk)
```

Comment:
What is current directory ``.``?  I think we shouldn't operate relative paths here.

Reply:
Changed to relative path